R/expectations-success.R

```diff
@@ -0,0 +1,25 @@
+#' Expect that code succeeds.
+#'
+#' Use it when failure should not abort the current test.
+#'
+#' @param expr Expression to evaluate
+#' @export
+#' @family expectations
+#' @examples
+#' expect_success(print("Hi!"))
+#' \dontrun{
+#' expect_success(stop("What?"))
+#' }
+expect_success <- function(expr) {
+  label <- find_expr("expr")
+  out <- evaluate_promise(expr)
```

Comment:
Good catch. I was wondering if this function necessary at all, we already have `expect_error(..., NA)`